src/applications/caregivers/sass/submitLoader.scss

```diff
@@ -0,0 +1,19 @@
+.loading-container {
```

Comment:
Added these styles to make `<p>` look like designs, and then needed to add a breakpoint to remove spacing for mobile.